### PR TITLE
DDF-3279 Remove TestEmbeddedSolr since we are testing that case elsewhere

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
This was a backport for https://github.com/codice/ddf/pull/2475
but ended up using a less invasive solution incase we want to turn the test back on in the future.

#### Choose 2 committers to review/merge the PR. 
@coyotesqrl 

#### How should this be tested? (List steps with links to updated documentation)
Successful build
#### What are the relevant tickets?
[DDF-3279](https://codice.atlassian.net/browse/DDF-3279)

### CODE REVIEW

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/ContainerPerClassItestSuite.java [hunk 1]
```diff
@@ -22,5 +21,5 @@
  * order as they appear in the array.
  */
 @RunWith(Suite.class)
-@Suite.SuiteClasses({TestEmbeddedSolr.class})
+@Suite.SuiteClasses({})
```
Comment (coyotesqrl) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/ContainerPerClassItestSuite.java]: ❓ No issue with having an empty suite? Does this pass at runtime okay? Does it still spin up the container and waste that time?

Comment (clockard) [reply distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/ContainerPerClassItestSuite.java]: This doesn't spin up the container so we save the time. It hasn't shown any issues running either.

**Pull request merged**